Sources/ComposableArchitecture/Store.swift

```diff
@@ -581,6 +581,7 @@ public typealias StoreOf<R: ReducerProtocol> = Store<R.State, R.Action>
   >: ReducerProtocol {
     let rootStore: Store<RootState, RootAction>
     let toScopedState: (RootState) -> ScopedState
+    private var parentStores: [Any]
```

Comment:
This is @tgrapperon's fix for #1379. It fixes the test in Release_StoreTests.swift.